packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -134,6 +134,14 @@ export interface CredentialsFromUsernameOptions {
    * @default - default master key
    */
   readonly encryptionKey?: kms.IKey;
+
+  /**
+   * The characters to exclude from the generated password.
+   * Has no effect if {@link password} has been provided.
+   *
+   * @default - the DatabaseSecret default exclude character set (" %+~`#$&*()|[]{}:;<>?!'/@\"\\")
```

Comment:
```suggestion
   * @default - the DatabaseSecret excludes character set (" %+~`#$&*()|[]{}:;<>?!'/@\"\\")
```

or something similar, feels like default is redundant as part of the `@default` text. few instances of these in the PR.

Reply:
It's a different 'default' word though - this says the the default for this is the same as the default for something else (in this case, `DatabaseSecret`).

Given this, are you OK with leaving this as-is?

Reply:
yep, seems reasonable! thanks for clarifying